pkg/session_recording/player.jsx

```diff
@@ -620,6 +620,229 @@
+                            259: 0.3700000000000002,
+                            260: 0.29000000000000015,
+                            261: 0.29000000000000015,
+                            262: 0.29000000000000015,
+                            264: 0.29000000000000015,
+                            268: 0.29000000000000015,
+                            270: 0.29000000000000015,
+                            273: 0.29000000000000015,
+                            276: 0.29000000000000015,
+                            282: 0.29000000000000015,
+                            289: 0.29000000000000015,
+                            291: 0.29000000000000015,
+                            295: 0.29000000000000015,
+                            306: 0.29000000000000015,
+                            310: 0.29000000000000015,
+                            314: 0.29000000000000015,
+                            316: 0.29000000000000015,
+                            318: 0.29000000000000015,
+                            321: 0.29000000000000015,
+                            323: 0.29000000000000015,
+                            325: 0.29000000000000015,
+                            331: 0.29000000000000015,
+                            335: 0.29000000000000015,
+                            338: 0.29000000000000015,
+                            340: 0.29000000000000015,
+                            341: 0.29000000000000015,
+                            343: 0.29000000000000015,
+                            344: 0.29000000000000015,
+                            346: 0.20000000000000007,
+                            348: 0.20000000000000007,
+                            349: 0.20000000000000007,
+                            351: 0.20000000000000007,
+                            353: 0.20000000000000007,
+                            355: 0.20000000000000007,
+                            356: 0.20000000000000007,
+                            357: 0.20000000000000007,
+                            360: 0.20000000000000007,
+                            365: 0.20000000000000007,
+                            371: 0.20000000000000007,
+                            375: 0.20000000000000007,
+                            378: 0.20000000000000007,
+                            383: 0.20000000000000007,
+                            388: 0.20000000000000007,
+                            392: 0.20000000000000007,
+                            403: 0.20000000000000007,
+                            409: 0.20000000000000007,
+                            414: 0.20000000000000007,
+                            418: 0.20000000000000007,
+                            424: 0.20000000000000007,
+                            423: 0.20000000000000007,
+                            422: 0.20000000000000007,
+                            421: 0.20000000000000007,
+                            419: 0.20000000000000007,
+                            420: 0.20000000000000007,
+                        };
```

Comment:
Could you please put this outside the function and class? Also, these seem to contain only a few significant digits. You can truncate them to just two digits after the point. E.g. 0.20, 0.29, 0.37, etc.

Reply:
Yes, sure, will do. Have you tried how it's working? Do you have any comments?